### DISCUSSION TITLE
Azure Cognitive Search

### DISCUSSION THREAD

**Discussion by tttll1091** [danny-avila/LibreChat#431] [Feature Requests & Suggestions]
https://github.com/openai/chatgpt-retrieval-plugin#azure-cognitive-search

Are you planning to implement this?

I hope it will be implemented in a pull request

**Comment by danny-avila** [comment]
Vector retrieval is planned, but it's not limited to azure cognitive search. I have to think about how to be inclusive with all the potential vector stores

**Comment by tttll1091** [comment]
If you want to search the company's data, it would be great if you could connect to the company's search engine.

In case of vector store, I think you need to upload the file each time
